### PR TITLE
Support ppc64le

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Review by bradrydzewski — APPROVED** [review]

**Comment by junaruga** [comment]
One comment about the keyword of `ppc64le` and `ppc64el`. both mean ppc64le little endian.
I guess that ppc64el is used in Debian based Linux distro, and ppc64le is used in Fedora/Red Hat based Linux distro.
https://wiki.debian.org/ppc64el

**Comment by junaruga** [comment]
Ubuntu
https://packages.ubuntu.com/hirsute/gcc
> ppc64el

Fedora's build system.
https://koji.fedoraproject.org/koji/buildinfo?buildID=1708867
> ppc64le

The `uname -m` might show the good candidate name for PPC little endian.

File: docker/Dockerfile.linux.ppc64le [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM alpine:3 as alpine
+RUN apk add -U --no-cache ca-certificates
+
+RUN wget https://github.com/isuruf/tmate/releases/download/2.4.0/tmate-2.4.0-static-linux-ppc64le.tar.xz
```
Comment (isuruf) [inline docker/Dockerfile.linux.ppc64le:4]: @bradrydzewski, FYI, this is from my github repo because there was no official release.

Comment (rpsene) [reply docker/Dockerfile.linux.ppc64le:4]: >     * Ubuntu: https://packages.ubuntu.com/hirsute/gcc => ppc64el
> 
>     * Fedora's build system. : https://koji.fedoraproject.org/koji/buildinfo?buildID=1708867 => ppc64le
> 
> 
> The `uname -m` might show the good candidate name for PPC little endian.

ubuntu@travis-bastion:~$ uname -a
Linux travis-bastion 4.15.0-137-generic #141-Ubuntu SMP Fri Feb 19 13:44:14 UTC 2021 ppc64le ppc64le ppc64le GNU/Linux

**Comment by bradrydzewski** [comment]
> One comment about the keyword of ppc64le and ppc64el. both mean ppc64 little endian.
I guess that ppc64el is used in Debian based Linux distro, and ppc64le is used in Fedora/Red Hat based Linux distro.

our standard is to use the same architecture name that is used by the Go compiler, which would be `ppc64le`

**Comment by junaruga** [comment]
OK. It's good to know it.

**Comment by isuruf** [comment]
ping on this

**Comment by isuruf** [comment]
Ping on this.

**Comment by ericpre** [comment]
Yes, please, that would be nice to get some progress on this front!

**Comment by marko-gacesa** [comment]
Thank you for your contribution @isuruf

**Pull request merged**